drools-parser/src/main/antlr4/org/drools/parser/DRLLexer.g4

```diff
@@ -32,6 +32,9 @@ WHEN : 'when';
 THEN : 'then';
 END : 'end';
 
+KWD_AND : 'and';
+KWD_OR : 'or';
```

Comment:
In order to avoid conflicts with JavaLexer `AND` (`&&`) and `OR` (`||`), I give the explicit names `KWD_AND` (`and`) and `KWD_OR` (`or`). If you have an idea of better names, let me know :)

Reply:
hi @tkobayas 
IMO "KWD_" is a good choice, knowing the contest. But I think it would be better to use the same prefix for all the "keywords" defined inside "DRLLexer.g4", to improve consistence. Moreover, inside "DRLLexer.g4" there are a lot of keywords defined and used elsewhere - I think prefixing all of them with "KWD_" would help a little bit... Just m2c....

Reply:
> prefixing all of them with "KWD_"

Thank you @gitgabrio  It's a good point. I checked the keywords in `DRLLexer.g4` and `JavaLexer.g4`. The duplicate keywords are `PACKAGE`, `IMPORT`, `STATIC`, `EXTENDS`, `SUPER` and their values are the same. So I'd say only `AND` and `OR` were the special cases (basically DRL keywords respect Java syntax). Of course, both Lexer files may be updated in the future so having "KWD_" for all keywords on DRL side would be a safeguard. The downside is that the DRL parser grammar could be a little busy... Hmm, may I postpone the decision for a while? I'd like to revisit later if we really want to prefix all keywords.

Reply:
:+1: TBH I was referring to our overall codebase; I mean, as "drools" developer I also have all the "drools repo" codebase in mind, and for example "GLOBAL" is defined also there... So the idea was to make strikingly evident that such keywords, wherever you read them, are not coming from "drools repo" itself but are specific of the drools-lsp module - I hope to have been clear, anyway you choose...

Reply:
Thanks @gitgabrio Not to forget, I filed a JIRA https://issues.redhat.com/browse/DROOLS-6999